spackbot/helpers.py

```diff
@@ -40,6 +51,20 @@ def temp_dir():
             os.chdir(pwd)
 
 
+def get_user_email(user):
+    """
+    Given a username, get the correct email based on creation date
+    """
+    response = requests.get("https://api.github.com/users/%s" % user).json()
```

Comment:
Agree that it's out of scope. I used requests, however, because using the github api handle didn't work for this simple request. It may have been that it didn't require authentication and the client always had it.